pkg/policy/policy_controller.go

```diff
@@ -159,7 +159,7 @@ func (pc *policyController) canBackgroundProcess(p kyvernov1.PolicyInterface) bo
 		val := os.Getenv("BACKGROUND_SCAN_INTERVAL")
 		interval, err := time.ParseDuration(val)
 		if err != nil {
-			logger.V(4).Info("failed to parse BACKGROUND_SCAN_INTERVAL env variable, falling to default 1h", "msg", err.Error())
+			logger.V(4).Info("BACKGROUND_SCAN_INTERVAL env variable is not set so using the default interval of 1h.", "msg", err.Error())
```

Comment:
I think it could be something as this:
```
The BACKGROUND_SCAN_INTERVAL env variable is not set, therefore the default interval of 1h will be used.
```

Reply:
@mohamedasifs123 can you address @MariamFahmy98 's comment ?